### PR TITLE
Fix sentiment example test

### LINKED ISSUE #7100: `test_thin_stack.py` is failing
The following test file in the sentiment analysis example fails. https://github.com/chainer/chainer/blob/master/examples/sentiment/test_thin_stack.py

It is a part of an example and is thus not tested by the CI. I have not yet investigated the cause nor the effects but the loss on the training data seems to decrease.

### PR DESCRIPTION
**Pull request by niboshi**
Fixes #7100

### CODE REVIEW

**Comment by hvy** [comment]
Thanks for the fix. Was wondering if you could you motivate the introduction of the new test directory?

**Comment by niboshi** [comment]
Simply it's inconvenience to be in separate directories if we're going to test it in CI.
That can be a separate PR if you prefer.

**Comment by hvy** [comment]
Thanks for the clarification. It sound to me like the scope is a bit widened although it makes perfect sense to run this test in the CI... How about discussing how to run them in the CI in a different issue, making this PR minimal?

**Comment by niboshi** [comment]
OK. Removed the move of the file.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Jenkins, test this please.

**Comment by hvy** [comment]
Failure is unrelated. LGTM!
